common/dynamicconfig/constants.go

```diff
@@ -2598,6 +2614,12 @@ const (
 	// Allowed filters: N/A
 	AllIsolationGroups
 
+	// DefaultIsolationGroupConfigStoreManagerGlobalMapping is the dynamic config value for isolation groups
+	// Note: This is not typically used for normal dynamic config (type 0), but instead
+	// it's used only for IsolationGroup config (type 1).
+	// KeyName: system.defaultIsolationGroupConfigStoreManagerGlobalMapping
+	DefaultIsolationGroupConfigStoreManagerGlobalMapping
```

Comment:
could you set it to a minus int so that if we iterate the key from Unknown to Last, this key won't be included.